doc/faq.rst

```diff
@@ -6,50 +6,20 @@
 Frequently Asked Questions
 ==========================
 
-1. About Pylint
-===============
-
-1.1 What is Pylint?
---------------------
-
-Pylint is a `static code checker`_, meaning it can analyse your code without
-actually running it. Pylint checks for errors, tries to enforce a coding
-standard, and tries to enforce a coding style.
-
-.. _`static code checker`: https://en.wikipedia.org/wiki/Static_code_analysis
-
-
-2. Installation
-===============
-
-2.1 How do I install Pylint?
-----------------------------
+How do I install Pylint?
+------------------------
 
 Everything should be explained on :ref:`installation`.
 
-2.2 What kind of versioning system does Pylint use?
----------------------------------------------------
+What kind of versioning system does Pylint use?
+-----------------------------------------------
 
 Pylint uses git. To get the latest version of Pylint from the repository, simply invoke ::
 
     git clone https://github.com/PyCQA/pylint
 
 .. _git: https://git-scm.com/
 
-2.3 What are Pylint's dependencies?
------------------------------------
-
-Pylint depends on astroid_ and a couple of other packages.
-See the following section for details on what versions of Python are
-supported.
-
-.. _`astroid`: https://github.com/PyCQA/astroid
-
-2.4 What versions of Python is Pylint supporting?
```

Comment:
I think it wouldn't hurt to keep this in the FAQ. As I can easily see people wondering about this and the FAQ would be one the first places I would look for this information.

Reply:
I'm going to remove the faq or create two faqs, one for users and one for developer. We might as well document the pylint version vs python interpreter... or maybe pip is handling that and you don't need to know ? Do you mind if we do that in the following merge request ?